explore/src/main/scala/explore/SideTabs.scala

```diff
@@ -33,26 +48,41 @@ object SideTabs {
         val tabsL = p.tabs.get.toNel
         val focus = p.tabs.get.focus
 
-        def tabButton(style: Css)(tab: AppTab): Button =
+        // group the tab buttons. Setting the value to 0 will
+        // cause the button to NOT be displayed.
+        def tabGrouping(tab: AppTab): Integer = tab match {
+          case AppTab.Overview       => 1
+          case AppTab.Observations   => 2
+          case AppTab.Targets        => 2
+          case AppTab.Configurations => 2
+          case AppTab.Constraints    => 2
+        }
```

Comment:
Let's put this info in `AppTab`, so that it's in the model and not in the display logic.

Let's also (eventually?) put there the condition required to display each tab (maybe it's just a list of roles, or a `User => Boolean`?). We can then filter here. We can use that info to validate when a user lands on a tab URL too. But I don't know if we are integrated with sso yet, so I wouldn't worry about this until we do have a model for users/roles.

For filtering, I **think** we can use `Zipper.unsafeSelect` traversal, or add a `filter` method to `Zipper`.

Reply:
Button grouping moved to AppTab. We will had filtering when we come to needing it.